DuckDuckGo/Bookmarks/Model/BookmarkDragDropManager.swift

```diff
@@ -196,6 +196,11 @@ final class BookmarkDragDropManager {
                 continue
             }
 
+            // Skip adding bookmark if the URL is already bookmarked
```

Comment:
I had to add it because otherwise dragging a favorite on the bookmark bar was causing a crash. Now that I think of it, it may have been an assertion failure though 🤔 will double check tomorrow and remove it if needed or try handling it better.

Reply:
Confirming it was the assertion failure in makeBookmark:
```swift
assert(url.absoluteString.lowercased() == url.absoluteString)
```
I've raised that with @tomasstrba separately. For now I'll remove this check here and let it fail on the assertion until that one is taken care of, because otherwise the behavior is correct and favorite doesn't get added to the bookmarks bar. Thanks again for spotting this, @jotaemepereira!